src/UI/Button/Button.tsx

```diff
@@ -0,0 +1,90 @@
+import style from "./Button.module.scss"
+import { Link } from 'react-router-dom';
+import { ReactNode } from 'react';
+
+import cnBind from 'classnames/bind'
+
+
+interface ButtonProps {
+    size?:'md' | 'sm' | 'xs',
```

Comment:
```suggestion
    size?: 'md' | 'sm' | 'xs'
```

Отступы
Запятые в типах и интерфейсах можно опустить